drupal-org.make

```diff
@@ -61,7 +61,7 @@ projects[commerce_cba][version] = 1.0-beta1
 projects[commerce_authnet][version] = 1.1
 projects[commerce_exactor][version] = 1.3
 projects[commerce_paymill][version] = 2.4
-projects[commerce_nosto_tagging][version] = 1.0
+projects[commerce_nosto_tagging][version] = 1.1
 projects[commerce_nosto_tagging][patch][] = https://drupal.org/files/issues/issue-2225883.patch
```

Comment:
Looks like this was committed? https://www.drupal.org/commitlog/commit/57373/2233e4fb7ebff02fabbd4ebba8192fb030b09193